srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Manager/Character/Entity/CharacterEntity.php

```diff
@@ -79,4 +100,16 @@ public function setName($name) {
         return $this;
     }
 
+    public function getRace() {
```

Comment:
write return type like  int, same for the functions below
```suggestion
    public function getRace(): int {
```